README.md

```diff
@@ -5,7 +5,7 @@
 This plugin transforms CSS using [PostCSS](https://github.com/postcss/postcss) and is part of the Skeletor ecosystem. To learn more about Skeletor, [go here](https://github.com/deg-skeletor/skeletor-core).
 
 ## Installation
-Install this plugin into your Skeletor-equipped project via the following terminal command: 
+Install this plugin into your Skeletor-equipped project via the following terminal command:
 ```
     npm install --save-dev git+https://git@github.com/deg-skeletor/skeletor-plugin-postcss.git
```

Comment:
Can you point this toward NPM? `npm install --save-dev @deg-skeletor/plugin-postcss`

Reply:
Yep!